modules/ptp-configuring-linuxptp-services-as-boundary-clock-dual-nic.adoc

```diff
@@ -0,0 +1,361 @@
+// Module included in the following assemblies:
+//
+// * networking/using-ptp.adoc
+
+:_content-type: PROCEDURE
+[id="ptp-configuring-linuxptp-services-as-bc-for-dual-nic_{context}"]
+= Configuring linuxptp services as a boundary clock for dual NIC hardware
+
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`) as boundary clock for dual NIC hardware by creating a `PtpConfig` custom resource (CR) object for each NIC.
+
+Dual NIC hardware allows you to connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding downstream clocks.
+
+[NOTE]
+====
+The following `PtpConfig` CRs configure PTP fast events by setting values for `ptp4lOpts`, `ptp4lConf` and `ptpClockThreshold`.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator.
+
+.Procedure
+
+. Create the following `PtpConfig` CR for the first NIC, in the `boundary-clock-ptp-config-nic1.yaml` file.
++
+[source,yaml]
+----
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: boundary-clock-ptp-config-nic1 <1>
+  namespace: openshift-ptp
+spec:
+  profile: <2>
+  - name: "profile1" <3>
+    ptp4lOpts: "-2 -s --summary_interval -4" <4>
+    ptp4lConf: | <5>
```

Comment:
Beginning around line 48, it might look nicer if all the values started at the same tab stop, provided it doesn't create any validation issues.

Reply:
Updated. I had thought of this previously, but was worried that the "tabbing" would add multiple extra space characters in the copied output. But it seems that preceding space characters are stripped, at least on the asciibinder build. Cool! I will do this. Makes it much more legible.